frontend/test/__support__/e2e/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -32,7 +34,7 @@ export function visitQuestionAdhoc(question, { callback, mode } = {}) {
  * @param {{database:number, table: number, mode: (undefined|"notebook"), limit: number, callback: function}} config
  */
 export function openTable({
-  database = 1,
+  database = SAMPLE_DB_ID,
   table,
   mode = null,
```

Comment:
20 lines below here, there's four `table: X`

Reply:
> Why not replacing the table references - or are you doing that in a separate PR? I'm guessing the IDs should be these: https://github.com/metabase/metabase/blob/master/frontend/test/__support__/sample_database_fixture.json#L219

Because I tried in and it breaks everything. Circular dependency.
Check the CI on this commit: https://github.com/metabase/metabase/pull/20729/commits/6f4d345bf5ab2b6d99ae4ac5d98cc147a57c1225

And then see this one as a solution: (https://github.com/metabase/metabase/pull/20729/commits/6319b9f888f25119592d791753e837d779a13820)